pkg/crc/machine/machine.go

```diff
@@ -682,3 +678,32 @@ func updateSSHKeyPair(sshRunner *crcssh.SSHRunner) error {
 
 	return err
 }
+
+func configureCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, pullSecret string) (rerr error) {
+	m := errors.MultiError{}
+	sd := systemd.NewInstanceSystemdCommander(sshRunner)
+	if _, err := sd.Start("kubelet"); err != nil {
+		return fmt.Errorf("Error starting kubelet: %s", err)
+	}
+
+	defer func() {
+		// Stop the kubelet service.
+		if _, err := sd.Stop("kubelet"); err != nil {
+			m.Collect(err)
+		}
+		if err := cluster.StopAndRemovePodsInVM(sshRunner); err != nil {
+			m.Collect(err)
+		}
+		rerr = m.ToError()
```

Comment:
wondering if we really need a MultiError here though .If sd.Stop() fails, it's very llikely cluster.StopAndRemovePodsInVM will fail too, so we could just report the sd.Stop() error in that case? And if sd.Stop succeeds, then we only have the StopAndRemovePodsInVM error to potentially report.